src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -84,7 +84,7 @@ export const styles = StyleSheet.create({
     fontSize: normalizeFontSize(14),
     fontFamily: fonts.inter[400],
     lineHeight: normalizeFontSize(16.94),
-    color: COLORS.black,
+    color: COLORS.gray500,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been changed from `COLORS.black` to `COLORS.gray500`. Similar to the previous comment, ensure that this color change aligns with the overall design and accessibility guidelines. Darker colors are generally more readable and accessible.
```diff
-    color: COLORS.black,
+    color: COLORS.gray500,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->